itez/beneficiary/models.py

```diff
@@ -49,15 +83,10 @@ class ServiceArea(models.Model):
     """
     Define service area properties.
     """
-    name = models.CharField(
-        _('Service Area'),
-        max_length=200
-    )
 
-    district = models.ForeignKey(
-        District,
-        on_delete=PROTECT
-    )
+    name = models.CharField(_("Service Area"), max_length=200)
```

Comment:
service area model now exists and we dont need it on beneficiary
the only linkage we need is district and service area (this one can be null and blank) but district should be there

we can get province via district and or service area if provided